GitHubAction/GitHubAction/ConsolePackagePresenter.cs

```diff
@@ -0,0 +1,111 @@
+﻿using Microsoft.Extensions.Logging;
+using Package.Builder;
+using Package.Domain.Exceptions;
+using Package.Domain.Models;
+using Package.Domain.Services;
+
+namespace GitHubAction;
+public class ConsolePackagePresenter : IPackagePresenter
+
+{
+    private readonly ILogger<ConsolePackagePresenter> _logger;
+
+    public ConsolePackagePresenter(ILogger<ConsolePackagePresenter> logger)
+    {
+        _logger = logger;
+    }
+
+    public void PresentUnauthorizedKey()
+    {
+        _logger.LogError("The provided key was not valid or authorized.");
+        Environment.Exit(403); // Forbidden
```

Comment:
Exit codes can't be larger then 255, changes this for all occurrences

Reply:
Also is it a good idea to let the presenter do this? Exiting after a failure seems to be more of an application responsibility. Presenter should only present the result in the correct way.